docs/architecture/011-capi-elements-as-components.md

```diff
@@ -0,0 +1,41 @@
+# Elements as Components
+
+## Context
+
+CAPI provides a representation of content which is strongly typed and like components.
+
+## Decision
+
+### CAPI Elements as Components
+
+The CAPI Blocks API exposes a suite of strongly typed elements. A component should be made for each element to render it.
+
+#### Elements Types
+
+Each element must have a type in dotcom rendering which specifies the data it may contain. It must have a `_type` field with a string literal of the elements name. This is to be used identifying which component to render.
+
+#### Unaltered Elements
+
+Some elements are represented in CAPI in a form which is directly renderable. For instance the `TextBlockElement`. These may be sent unaltered to dotcom rendering.
+
+Their `_type` field must remain unaltered from the CAPI response. They should be of the form `model.liveblog.`.
+
+### Altered Elements
+
+Some elements contain data which is unsuitable for direct rendering, these should be transformed outside of dotcom rendering. Their `_type` field should be set and should read `dotcom.element.`.
+
+An element should not merely contain an `html` field. Except in the cases where any other representation would be equivalently complex in representation. This may be used as a stop gap solution.
```

Comment:
I don't think I understand this para. Are you suggesting an altered element should contain fields other than `html`?